dlf/plugins/oai/class.tx_dlf_oai.php

```diff
@@ -967,13 +967,9 @@ protected function verbIdentify() {
 
 		} else {
 
-			if (TYPO3_DLOG) {
-
-				t3lib_div::devLog('[tx_dlf_oai->verbIdentify()] Incomplete plugin configuration', $this->extKey, SYSLOG_SEVERITY_ERROR);
-
-			}
+			t3lib_div::devLog('[tx_dlf_oai->verbIdentify()] Incomplete plugin configuration', $this->extKey, SYSLOG_SEVERITY_ERROR);
```

Comment:
Why did you remove the check for `TYPO3_DLOG`? I think Goobi.Presentation should only post messages to devLog if an administrator has explicitly turned logging on by setting `TYPO3_DLOG` in the TYPO3 configuration. At least this is how I understood the purpose of this setting.